Automattic/wp-calypso
# GH Integration: Add card to display list of repositories and branches to select from.

Closes https://github.com/Automattic/wp-calypso/issues/73404 and part of https://github.com/Automattic/wp-calypso/issues/70378.

#### Proposed Changes

* Add card to display list of user's github repositories and branches that may select from when connecting their site to github.

#### Testing Instructions

See pet6gk-5n-p2. You need to patch your sandbox with D101662-code for the search to work.

**Pre-requisites** 

```
- Sandbox the endpoints from this [diff](D100978-code)
```

- On any AT sites go to the hosting page
- Scroll down the github connect branch section
- A card should be displayed with your list of repositories
- Verify changing repo updates branches
- Verify display on smaller screens
- Verify it renders correctly on RTL languages

![image](https://user-images.githubusercontent.com/47489215/217135696-9d524479-c873-40f8-9f87-18ac6adb3605.png)



*

#### Pre-merge Checklist





Related to https://github.com/Automattic/wp-calypso/issues/70378
